apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx

```diff
@@ -124,7 +124,7 @@ export default function SandboxPullRequestConversationPage() {
   useEffect(() => {
     setActivities(activityData)
   }, [activityData])
-  const currentUser = currentUserData?.display_name || ''
+  const currentUser = useMemo(() => currentUserData?.display_name, [currentUserData?.display_name])
```

Comment:
Check out this link regarding above change: https://chatgpt.com/share/672dd3aa-c7dc-8005-a5d2-5040c58ac46b

Reply:
initially it was that but it didn't change it @vardanbansal-harness